scripts/globals/bluemagic.lua

```diff
@@ -9,186 +9,28 @@ TPMOD_ACC = 3
 TPMOD_ATTACK = 4
 TPMOD_DURATION = 5
 
+-- The SC the spell makes
```

Comment:
These should not exist in a global space unattached to any object.  They were removed initially because they also conflict with weaponskill definitions.  If really needed as a global, they should be tabled and attached to the xi global.

Reply:
Do you want all of them gone or just TPMOD ones?

Reply:
At a future date, we may want blue spells to have their properties tabled. For now, though, it isnt worse than what we have now.